recipes/libcf/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "libcf" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}-beta11.tar.gz
+  url: http://uvcdat.llnl.gov/cdat/resources/{{ name }}-{{ version }}-beta11.tar.gz
+  sha256: e40d3826ca2b47538b075f69f716798e255fe396fd3ae68e8b051be57a951bb4
+
+build:
+  number: 1
+  skip: True  # [win or py3k] 
+
+requirements:
+  build:
+    - toochain  # [osx]
+    - python
+    - hdf5 1.8.17|1.8.17.*
+    - libnetcdf 4.4.*
+    - lapack >=3.4.2
+    - clapack >=3.2.1
+    - curl
+    - ossuuid
+    - gcc
```

Comment:
Do you need `Fortran` or `libgmop` for this package? If not you don't need the `gcc` package.